pages/exercises/[lessonSlug].tsx

```diff
@@ -45,14 +80,113 @@ const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
     { text: 'exercises', url: `/exercises/${currentLesson.slug}` }
   ]
 
+  const exercise = mockExercises[exerciseIndex]
+
   return (
     <Layout title={currentLesson.title}>
```

Comment:
Do you think the admins should be the only ones with access to this page, or does it matter since there aren't links that users can use to access this page?

Reply:
It shouldn't matter since there aren't any links that point to this page.